charts/v2.0.0-alpha.1/azuredisk-csi-driver/templates/csi-azuredisk-controller.yaml

```diff
@@ -180,6 +180,7 @@ spec:
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--metrics-address=0.0.0.0:{{ .Values.controller.metrics.port }}"
             - "--is-controller-plugin=true"
+            - "--enable-perf-optimization={{ .Values.linux.enablePerfOptimization }}"
```

Comment:
I don't see a corresponding change to `Values.yaml`. Did you forget to add the file?

Reply:
That setting is already there in yaml file, but right now it was only for nodeserver, since that's the only place we were doing any validation/perfProfile related stuff.
Now that I am doing some validation in controller (CreateVolume), I need to put things behind this flag, just so we can disable the code path if need be.